code/api/src/modules/user/mutations.js

```diff
@@ -3,7 +3,7 @@ import { GraphQLString, GraphQLInt } from 'graphql'
 
 // App Imports
 import { UserType } from './types'
-import { create, remove } from './resolvers'
+import { create, remove, update } from './resolvers'
```

Comment:
Then destroy next?

Reply:
Actually, the destroy function just goes straight to the resolvers and skips over mutations. I'm glad to explain more